templates/server/config.ru.erb

```diff
@@ -1,5 +1,7 @@
-### Next part of the file is managed by a different template ###
-## Module:           '<%= scope.to_hash['module_name'] %>'
+#
+# WARNING: This file is auto-generated by puppet!
+# WARNING: To change it, edit the file in puppet/templates/server/config.ru.erb
```

Comment:
I'm really unsure if it's a good idea to tell users to edit files... They should rather set parameters to chnage the generated files, IMHO.

Reply:
Good point, i've removed that line.